app/controllers/business/orders_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
   end
 
   def show
+    @business = Business.find(slug: params[:slug])
```

Comment:
Fixed found an issue with edit business that is now resolved. Also added a button for it to work.